lib/helper/WebDriverIO.js

```diff
@@ -1496,6 +1736,53 @@ class WebDriverIO extends Helper {
     });
   }
 
+  /**
+   * Switch focus to a particular tab by its number. It waits tabs loading and then switch tab
+   */
+  changeTab(num, amountOfTabs = null, sec = null) {
```

Comment:
sorry, the behavior is not simple to get. WHy should I declare the `amountOfTabs`? It is not even documented properly.
I'd recommend to use 2 methods:

```js
I.switchToNextTab(); // switch to next tab
I.switchToNextTab(2); // switch to tab +2 from current
```
In code:

```js
switchToNexTab(num = 1, sec = null) {
```

In the same manner `switchToPreviousTab` can be implemented